app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.health = health
+        self.name = name
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return f"{{Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(animal_to_bite: Herbivore) -> None:
+        is_it_carnivore = not isinstance(animal_to_bite, Carnivore)
+        if animal_to_bite.hidden is False and is_it_carnivore:
```

Comment:
There is no need for the `is_it_carnivore` variable. Add it to the if condition directly. Besides that, evaluating the `.hidden` attribute with `is` is over-explicit here, just assume that it will always have a boolean value and build this if condition using `not`.

Reply:
if i don`t use is_it_carnivore variable,then flake8 will that "line is to long" . Same with return f-string in 16 line